cockroachcloud/serverless-movr.md

```diff
@@ -0,0 +1,125 @@
+get-the-cluster-connection-string---
```

Comment:
'get-the-cluster-connection-string' prior to the '---' that starts the page's YAML front matter appears to be a copy paste error - pretty sure this is breaking the build

Reply:
oh! thank you. no idea how that got there.